static/app/components/autoSelectText.tsx

```diff
@@ -1,4 +1,4 @@
-import {CSSProperties} from 'react';
+import {CSSProperties, useRef} from 'react';
```

Comment:
Can't we just use `React.CSSProperties` without the import?

Reply:
@EvanPurkhiser why do you think it's better to use `React.CSSProperties` ?

Reply:
Because then you don't have to import it :) the `React` type namespace is in the global scope!